pkg/api/apierrors/kinds.go

```diff
@@ -46,6 +46,14 @@ func (n Name) HTTPStatus() int {
 type Kind struct {
 	Name   Name   `json:"name"`
 	Reason string `json:"reason"`
+
+	IsSkipLoggingToExternalService bool `json:"-"`
```

Comment:
In authui, the APIError could be read from cookie, thus deserialized from a serialized APIError. So I think IsSkipLoggingToExternalService could be lost.

Reply:
The existing usage of IsSkipLoggingToExternalService are errors unrelated to authui, so this is not a problem at the moment.

I think we will not serialize this private field. In case in the future, we really need to support IsSkipLoggingToExternalService, then I think we should log the error at the time of writing the cookie.